src/swagger_output.json

```diff
@@ -0,0 +1,105 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "v1.0.0",
+    "title": "Brahm Bunga Dodra Swagger",
+    "description": "Implementation of Swagger with TypeScript"
+  },
+  "servers": [
+    {
+      "url": "http://localhost:5050",
+      "description": ""
+    }
+  ],
+  "paths": {
+    "/settings/": {
```

Comment:
Check if swagger_output.json is needed

Reply:
Deleted the swagger_output.json file. Also made changes to format of the response of each endpoint.